File: Logick/ViewModel/PlayerInGameView.cs
using BlackJack.DataAccess.Entities.Enums;
using System.Collections.Generic;


namespace BlackJack.BusinessLogic.ViewModel
{

    public class PlayerInGameView
    {
        public long PlayerId { get; set; }
        public PlayerType PlayerType { get; set; }
        public string PlayerName { get; set; }
        public List<CardView> Cards { get; set; }
        public int Points { get; set; }
        public PlayerStatus PlayerStatus { get; set; }
        public long GameId { get; set; }

        public PlayerInGameView()
        {
            PlayerStatus = PlayerStatus.Play;
            Cards = new List<CardView>();
        }
    }
}

File: BlackJack.Api/UI/src/app/shared/models/player-in-game-view.ts
import { CardView } from './card-view';

export interface PlayerInGameView {
  playerInGameViewItems: PlayerInGameViewItem[];
}

export interface PlayerInGameViewItem {
  playerName: string;
  cards: CardView[];
  points: number;
  playerStatus: string;
  gameId: number;
}
